docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -63,3 +63,15 @@ For example:
 
 The same applies to `slotProps` and `componentsProps`.
 :::
+
+### Replace `defaultCalendarMonth` with `referenceDate`
+
+The `defaultCalendarMonth` has been removed in favor of the more flexible `referenceDate` prop.
+It was available in `DateCalendar` and all the pickers using it for date editing.
```

Comment:
```suggestion
```

Is it necessary? People already using it are not concerned by this change

Reply:
My goal here was to help people know if they can skip this section

Reply:
Removed